config/examples/sandbox_proxy.yml

```diff
@@ -5,7 +5,8 @@ base: &default
   ignore_test_failures: []
   hosts:
     - localhost:8091
-  backend_host: backend:3000
+  # Backend endpoint as visible by apicast
+  backend_host: localhost:3001
   shared_secret: dev
```

Comment:
As we're at it, how about removing `shared_secret`, `nginx_port` and `hosts`? They are not used, according to my prior checks.

Reply:
Are these some leftovers from apicast v1?

I removed them anyway.

Reply:
Yeah, some old stuff... 
`shared_secret` and `hosts` were used by some `Apicast::Sandbox`, but it was removed here: https://github.com/3scale/porta/pull/2538/files

And `nginx_port` last reference was removed here  https://github.com/3scale/porta/pull/2539/files